src/routes/v2/authenticated/sites.ts

```diff
@@ -188,6 +195,40 @@ export class SitesRouter {
     return res.status(404).json({ message: "Unable to get staging status" })
   }
 
+  getLinkCheckerStatus: RequestHandler<
+    { siteName: string },
+    BrokenLinkErrorDto | ResponseErrorBody,
+    never,
+    never,
+    { userWithSiteSessionData: UserWithSiteSessionData }
+  > = async (req, res) => {
+    const { userWithSiteSessionData } = res.locals
+    const result = await this.repoCheckerService.readRepoErrors(
+      userWithSiteSessionData.siteName
+    )
+    if (result.isOk()) {
+      return res.status(200).json(result.value)
+    }
+    return res.status(400).json({ status: "error" })
```

Comment:
Is this really a 400 status? This seems to be a catch all instead of error handling we can send to the FE

Reply:
What are the possible failure modes in this?

Reply:
good point, seems like 404 is more semantically meaninful here. ie the list of possible errors that is supposed to exist is not found. 

failure modes -> multiple, but the as of the current state, there is only one recovery path (eg, by re-reunning the checker again)

Reply:
[d1d1814](https://github.com/isomerpages/isomercms-backend/pull/1148/commits/d1d181420865d3535d44d952a88589ce12cb9047) changed here